.travis.yml

```diff
@@ -264,6 +264,12 @@ os: linux
     cache:
       directories:
         - "$HOME/.ccache"
+        - depends/built
+        - depends/sources
+        - depends/work
+        - depends/x86_64-linux-gnu
+        - depends/x86_64-unknown-linux-gnu
+        - depends/x86_64-w64-mingw32
```

Comment:
For the sake of completeness it would be great to also include the paths for MacOS. I'm not 100% sure that caching the depends folder is safe though, there are circumstances which might require a rebuild from scratch:
- Changes to OS libraries in the docker images that we dynamically link against, e.g. libc6/glibc
- Changes to package versions in depends/packages/*.mk, the build system should detect this. But a manual test to verify this would be great.

Reply:
This caching directives are for linux OS ... OSX images need different directives and I wouldn' focus too much for the moment on a build which relies on outdated Xcode